docs/vagrant.md

```diff
@@ -0,0 +1,132 @@
+
+Although Vagrant supports [VirtualBox](https://www.virtualbox.org/), [Hyper-V](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v), [Docker](https://www.docker.com/), or [VMware Fusion](https://customerconnect.vmware.com/downloads/get-download?downloadGroup=FUS-PUBTP-2021H1), `VirtualBox` is the only environment that has been developed and tested thus far. If you have need to use Hyper-V, Docker, or VMware, please create an issue and request a new feature.
+
+## Notes
+
+This was tested using:
+
+- Vagrant v2.4.0
+- VirutalBox v7.0.12
+
+## Prerequisites
+
+1. You will require 4 GB of memory.
+1. You will require 8 GB of disk space.
+1. Ports 4200, 4300, and 5432 must be available on host environment. If you have powershell installed you can use this command to list active ports:
+
+   ```powershell
+   Get-NetTCPConnection | Where-Object { $_.State -eq 'Listen' }
+   ```
+
+1. `npm install` creates symbolic links. As a result, the VM user: `vagrant` requires `create symbolic link` authority. In some environements, only the administrator has this authority. To resolve this issue in Windows, the following steps can be performed:
+   1. Launch the `Local Security Policy` app from the `start menu` (or `Win+R`, then type `secpol.msc`)
+   1. Navigate to `Local Policies` -> `User Rights Assignment`.
+   1. Open the `Create symbolic links` property. By default it has only `Administrators` listed.
+   1. Click `Add User or Group...` button and type `Authenticated Users` in the `Enter the object names to select` field and then press the `OK` button. This will grant `symbolic link` priviledge to all users who log on with credetials.
+   1. Close any open windows by clicking `OKs` and and then `log off` and `log on` to activate.
+   1. Symbolic links should now work in VirtualBox shared folders without administrator priviledges.
+
+## Setup Steps
+
+1. [Download](https://developer.hashicorp.com/vagrant/downloads?ajs_aid=ba208f95-d5a1-457d-abeb-49d458b95eec&product_intent=vagrant) Vagrant software.
+1. Reference these [installation](https://developer.hashicorp.com/vagrant/docs/installation) instructions.
+1. Verify installation by running `vagrant -v` in a terminal session.
+1. Install [VirtualBox](https://www.virtualbox.org/).
+1. If you have not already done so, fork repo to your local machine.
+1. Vagrant uses a special configuration file called `Vagrantfile` that contains all the information required for the creation of a Virtual Machine (guest) on your workstation (host) . It is located n your repo root directory.
+1. To start the creation of your virtual environment, run this command in the repo root directory. (This takes a while so be patient.)
+
+   ```bash
+   vagrant up
+   ```
+
+1. Once the `vagrant up` command completes, you can logon into your virutal machine using this command. (password: `vagrant`)
```

Comment:
```suggestion
1. Once the `vagrant up` command completes, you can logon into your virtual machine using this command. (password: `vagrant`)
```

I would recommend installing the "Code Spell Checker" extension in VS Code, so it is easier to catch these issues.

Reply:
done